Joplin_install_and_update.sh

```diff
@@ -205,9 +205,15 @@ if command -v lsb_release &> /dev/null; then
   # Check for "The SUID sandbox helper binary was found, but is not configured correctly" problem.
   # It is present in Debian 1X. A (temporary) patch will be applied at .desktop file
   # Linux Mint 4 Debbie is based on Debian 10 and requires the same param handling.
-  if [[ $DISTVER =~ Debian1. ]] || [ "$DISTVER" = "Linuxmint4" ] && [ "$DISTCODENAME" = "debbie" ] || [ "$DISTVER" = "CentOS" ] && [[ "$DISTMAJOR" =~ 6|7 ]]
+  #
+  # This also works around Ubuntu 23.10+'s restrictions on unprivileged user namespaces. Electron
+  # uses these to sandbox processes. Unfortunately, it doesn't look like we can get around this
+  # without writing the AppImage to a non-user-writable location (without invalidating other security
+  # controls). See https://discourse.joplinapp.org/t/possible-future-requirement-for-no-sandbox-flag-for-ubuntu-23-10/.
+  if [[ $DISTVER = "Ubuntu23.10" || $DISTVER =~ Debian1. || ( "$DISTVER" = "Linuxmint4" && "$DISTCODENAME" = "debbie" ) || ( "$DISTVER" = "CentOS" && "$DISTMAJOR" =~ 6|7 ) ]]
   then
     SANDBOXPARAM="--no-sandbox"
+    print "${COLOR_YELLOW}WARNING${COLOR_RESET} Electron sandboxing disabled. Avoid loading untrusted notes/PDFs/audio/video files."
```

Comment:
It may also make sense to link to a forum post (or FAQ entry) that gives a high-level summary of what this means (and how it affects Joplin's security).

Reply:
Yes, such info is always helpful and might put people at ease. However, I am not sure if we already have any forum posts or a FAQ entry in the docs.

Reply:
I think a link would be better and the message "Avoid loading untrusted notes/PDFs/audio/video files" could be removed because that's a bit vague and probably always true.

But a link with more information would clarifies the issue.

Reply:
@personalizedrefrigerator, could you link to one of the existing forum posts? I'm not sure which one is the best. There's no need to create a FAQ entry for now

Reply:
I've added a link to https://discourse.joplinapp.org/t/possible-future-requirement-for-no-sandbox-flag-for-ubuntu-23-10/32160.

Of the [search results for `no-sandbox`](https://discourse.joplinapp.org/search?q=no-sandbox) on the forum, it and [a snap-related post](https://discourse.joplinapp.org/t/joplin-desktop-launch-with-no-sandbox-flag/29761) seemed most relevant.

Reply:
Thanks, could you set it to `https://discourse.joplinapp.org/t/32160/5` please? As it's less likely to wrap in the terminal